lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -142,7 +144,9 @@ def set_up_probability_above_threshold_spot_cube(
     cube.add_aux_coord(AuxCoord(np.linspace(120, 180, x_dimension_length,
                                             dtype=np.float32),
                                 'longitude', units='degrees'), data_dims=2)
-    cube.attributes["relative_to_threshold"] = "above"
+    cube.coord(
+        var_name="threshold"
+    ).attributes["spp__relative_to_threshold"] = "above"
```

Comment:
This indentation looks wrong...but that could just be me.

Reply:
I've edited the indentation. Is that better?

Reply:
Let's say yes.